Template/user/sidebar.php

```diff
@@ -1,5 +1,5 @@
-<?php if ($this->user->isAdmin()): ?>
-    <li <?= $this->app->getRouterController() === 'timetable' ? 'class="active"' : '' ?>>
-        <?= $this->url->link(t('Manage timetable'), 'timetable', 'index', array('plugin' => 'timetable', 'user_id' => $user['id'])) ?>
-    </li>
-<?php endif ?>
\ No newline at end of file
+<?php if ($this->user->isUser()): ?>

```

Comment:
yes.but it may generate error when navigate without login in to kanboard. that's why i checked any type of user logged in.also can use isLoggedin() function.